lib/pinyin/zhuyin_parsers.ex

```diff
@@ -0,0 +1,118 @@
+defmodule Pinyin.ZhuYinParsers do
+
+  import NimbleParsec
+  alias Pinyin.Parsers.Utils, as: Utils
+
+  # Empty tone must be last for parsing to work
+  zhuyin_tones = ["˙", "ˊ", "ˇ", "ˋ", ""]
+
+  initials = ~w(
+    ㄅ ㄆ ㄇ ㄈ
+    ㄉ ㄊ ㄋ ㄌ
+    ㄍ ㄎ ㄏ
+    ㄐ ㄑ ㄒ
+  )
+
+  standalone_initials = ~w(
+    ㄓ ㄔ ㄕ ㄖ
+    ㄗ ㄘ ㄙ
+  )
+
+  two_finals = ~w(
+    ㄧㄚ
+    ㄨㄚ
+    ㄧㄥ
+    ㄧㄤ
+    ㄧㄝ
+    ㄨㄛ
+    ㄨㄥ
+    ㄨㄤ
+    ㄧㄠ
+    ㄨㄞ
+    ㄩㄝ
+    ㄩㄥ
+    ㄧㄡ
+    ㄨㄟ
+    ㄩㄝ
+    ㄧㄢ
+    ㄨㄢ
+    ㄩㄢ
+    ㄧㄣ
+    ㄨㄣ
+    ㄩㄣ
+  )
+  single_finals = ~w(
+    ㄧ ㄨ ㄩ
+    ㄚ ㄛ ㄜ ㄦ
+    ㄞ ㄟ ㄠ ㄡ
+    ㄢ ㄣ ㄤ ㄥ
+  )
+
+  initials_parser =
+    Enum.concat(initials, standalone_initials) |> Utils.wordlist_to_parser
```

Comment:
Here and in subsequent lines, I would add parens after the call. As far as I know, this is done for most Elixir code (I think `mix format` might do this for you)